aptos-move/framework/aptos-token/sources/property_map.move

```diff
@@ -160,6 +162,35 @@ module aptos_token::property_map {
         *property_val = value;
     }
 
+    public fun create_property_value_raw(
+        value: vector<u8>,
+        type: String
+    ): PropertyValue {
+        PropertyValue {
+            value,
+            type,
+        }
+    }
+
+    /// create a property value from generic type data
+    public fun create_property_value<T: copy>(data: &T): PropertyValue {
```

Comment:
Can we add a unit test for this?

Reply:
I think we don't need this if we can simplify the logic.